lib/mark-optional/index.ts

```diff
@@ -1,5 +1,6 @@
 import { OptionalKeys } from "../optional-keys";
+import { Prettify } from "../prettify";
 
 export type MarkOptional<Type, Keys extends keyof Type> = Type extends Type
-  ? Partial<Type> & Required<Omit<Type, Keys | OptionalKeys<Type>>>
+  ? Prettify<Partial<Type> & Required<Omit<Type, Keys | OptionalKeys<Type>>>>
```

Comment:
I'd like to challenge your decision and ask what's the point of doing `Prettify` when your type is very complicated? On really big projects I don't think it makes much sense. It will be unreadable and cut in half. This type is available so user can manually apply it if it's useful, but I don't think we need to make this decision here and in other utility functions

Reply:
I still believe there’s value in using `Prettify` with types like `Mark*`, simply because it doesn’t degrade the experience—at worst, it’s neutral (e.g., when the display gets trimmed), and at best, it improves readability.

Btw, `type-fest` also prettifies [their version of `Mark*` types](https://github.com/sindresorhus/type-fest/blob/main/source/set-optional.d.ts#L30).

@Beraliv Do you see any potential downsides of using `Prettify` here?

Reply:
@som-sm agree to disagree, but let's find out about it. Worst case scenario - it won't be useful, it will break something serious so we will roll it back, document that X didn't work out because of (1), (2), etc. So I'm happy for you to keep `Prettify`